docfx/en/components/toc.json

```diff
@@ -121,13 +121,13 @@
   },
   {
     "note": "THIS MAIN NODE FOR NEW XPLAT GRID",
-    "exclude": ["Angular", "React"],
+    "exclude": ["Angular"],
     "name": "Grids & Lists",
     "header": true,
     "href": "grids/grids-header.md"
   },
```

Comment:
The grid component for React is relatively done. There is more testing and bug fixing needed but we decided to start working on the documentation bit by bit in parallel. The changes in toc.json are needed otherwise the build would fail due to hyperlink errors.
The case with the samples is a little more complex and PRs with samples to [igniteui-react-examples](https://github.com/IgniteUI/igniteui-react-examples) will come further down the road. @dkamburov would be able to shed more light on that.
I am converting this PR to draft as it is still WIP and is not for merging yet.